### ISSUE TITLE
[SUGGESTION] Read the PHP Doc Block and Parse the parameter and Comment when auto complete

### ISSUE THREAD

**Issue by benyaminl** [HvyIndustries/crane#363]
Is ther any plan to make crane could Read the PHP Doc Block and Parse the parameter and Comment when ctrl+space while code (Code Completation or such)?
Because It's pain in the ass sometimes to see the or peeks too much, could be better show it when directly on auto completation.. Sorry for bad english

**Comment by nevadascout** [comment]
Thanks for the suggestion!

This is actually something I've been testing on an alpha build for the last few weeks. Hopefully I can make this available to everyone fairly soon!

**Comment by PapsOu** [comment]
@nevadascout Do you have planned to release this most wanted feature soon ?

**Comment by nevadascout** [comment]
Yes! I've been testing it for some time and it seems to work quite well.

I have a few bugs to squash and then I will release it -- I would guess within 7 days.
